include/franka_plan_runner.h

```diff
@@ -100,15 +100,21 @@ class FrankaPlanRunner {
       joint_position_callback_;
 
   // keeping track of time along plan:
-  double franka_time_;
+  double franka_time_ {};
   // pause related:
   utils::RobotStatus status_;
   long timestep_ = 1;
   float target_stop_time_;
   float stop_duration_;
   float stop_margin_counter_ = 0;
-  int cur_plan_number_ = -1;               // for ensuring the plan is new
-  const double lcm_publish_rate_ = 200.0;  // Hz
+  int cur_plan_number_ = -1;  // for ensuring the plan is new
+
+  // Franka supports up to 1 kHz, but here we use 200 Hz, because cobots
```

Comment:
This comment is somewhat misleading - there is a clear distinction that should be made between control frequency and publish frequency. My understanding is we control at 1kHz but only publish status at 200Hz. Please clarify this in the comment